components/net/filemanager_thread.rs

```diff
@@ -189,23 +189,16 @@ impl<UI: 'static + UIProvider> FileManager<UI> {
                         spawn_named("dec ref".to_owned(), move || {
                             // Since it is simple DecRef (possibly caused by close/drop),
                             // unset_url_validity is false
-                            let _ = sender.send(store.dec_ref(&id, &origin, false));
+                            let _ = sender.send(store.dec_ref(&id, &origin));
                         })
                     } else {
                         let _ = sender.send(Err(BlobURLStoreError::InvalidFileID));
                     }
                 }
-                FileManagerThreadMsg::IncRef(id, origin) => {
```

Comment:
No need to explicitly `IncRef`. I now know why: because all cross-process reference is created by construction (through `Select` and `AddSlicedURLEntry`), so it should be natural not to do `inc_ref` on the script side.